src/__test__/selections-factory.spec.js

```diff
@@ -0,0 +1,126 @@
+import { getCellStyle, selectCell } from '../selections-factory';
+
+describe('selections-factory', () => {
+  // describe('initSelections', () => {
```

Comment:
I can't run this since the nebula hooks running in `initSelections` needs a visualization component. Worth it to mock one just to test? Would be one of those tests that look good but doesn't really test much.